src/USER-PLUMED/Install.sh

```diff
@@ -0,0 +1,56 @@
+# Install/unInstall package files in LAMMPS
+# edit 2 Makefile.package files to include/exclude ATC info
+
+if (test $1 = 1) then
+
+  if (test -e ../Makefile.package) then
+    sed -i -e 's|^PKG_LIB =[ \t]*|& $(PLUMED_LOAD) |' ../Makefile.package
+    if ( ! test -e ../../lib/plumed/plumed2*/src/lib/Plumed.inc.static ) then
+        sed -i -e 's|^PKG_SYSINC =[ \t]*|& -D__PLUMED_HAS_DLOPEN |' ../Makefile.package
+    fi
+  fi
+
+  if (test -e ../Makefile.package.settings) then
+    # This is for statically linking plumed2
+    if ( test -e ../../lib/plumed/plumed2*/src/lib/Plumed.inc.static ) then  
```

Comment:
Using wildcards in this way can break rather easily and thus should be avoided if possible. Better to use symlinks in the lib/plumed folder to point to a specific version and/or an installed version as it is done by latte or voronoi for example.

Reply:
This code is also completely rewritten and adapted to be more like it is for other LAMMPS packages. The processing of the plumed specific settings is now handled by `lib/plumed/Install.py` and results in an automatically generated customized `lib/plumed/Makefile.lammps`.